apps/dates/__init__.py

```diff
@@ -0,0 +1 @@
+from .fakers import HistoricDateTimeProvider
```

Comment:
Looks like this breaks building the Django image outside of dev environment, since `factory-boy` is a dev-only dependency